src/utils/tmi-api.js

```diff
@@ -0,0 +1,47 @@
+
+const ENDPOINT = 'https://tmi.twitch.tv/';
+
+function get(path, options) {
+    return fetch(`${ENDPOINT}${path}`, options);
```

Comment:
you need to make sure fetch is being polyfilled, otherwise just rely on jquery's ajax for now until all the api utils get upgraded